### DISCUSSION TITLE
Making more typefaces from Google Fonts easily available

### DISCUSSION THREAD

**Discussion by pglevy** [uswds/uswds#4956] [General]
I was thinking that if it were easier to "plug and play" some curated custom typefaces, we might see some more typographic diversity in government sites. So I started a proof-of-concept repo to see what it would take to compile what you'd need to use more options from Google Fonts. There are a few steps involved, like measuring cap heights, converting web fonts, setting up theme tokens, etc. But once this is done, others can simply copy and paste the token settings and font files and they're good to go.

Check it out here and let me know if you think this would be helpful to continue…
https://github.com/pglevy/uswds-custom-theme-typefaces

**Comment by mejiaj** [comment]
Thanks for sharing @pglevy! 

We've been working on our custom font guidance as well and have an issue to bring that guidance to the website https://github.com/uswds/uswds-site/issues/1783.

**Reply by pglevy** [reply]
Glad to hear it! Is that going to include mainly guidance or additional pre-configured fonts as well? Anyway, I'm thinking about opening this repo up as a [Hacktoberfest](https://hacktoberfest.com/) project to see if I can't get more designers involved in making contributions.
